integrations/visual_studio/cmake.rst

```diff
@@ -0,0 +1,196 @@
+Conan **profiles** are highly recommended.
+
+Using cmake-conan
+-----------------
+
+The **cmake-conan** project in https://github.com/conan-io/cmake-conan is a CMake script that runs an ``execute_process`` that automatically
+launches :command:`conan install` to install dependencies. The settings passed in the command line will be derived from the current CMake
+configuration, that will match the Visual Studio one. This script can be used to further automate the installation task:
+
+.. code-block:: cmake
+
+    project(Example CXX)
+    cmake_minimum_required(VERSION 2.8.12)
+
+    # Download automatically, you can also just copy the conan.cmake file
+    if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+    message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
+        file(DOWNLOAD "https://raw.githubusercontent.com/conan-io/cmake-conan/v0.9/conan.cmake"
+                    "${CMAKE_BINARY_DIR}/conan.cmake")
+    endif()
+
+    include(${CMAKE_BINARY_DIR}/conan.cmake)
+
+    conan_cmake_run(CONANFILE conanfile.txt
+                    BASIC_SETUP)
+
+    add_executable(example example.cpp)
+    target_link_libraries(example ${CONAN_LIBS})
+
+This code will manage to download the **cmake-conan** CMake script, and use it automatically, calling a :command:`conan install` automatically.
+
+There could be an issue, though, for the ``Release`` configuration. Internally, the Visual Studio 2017 defines the ``configurationType`` As
+``RelWithDebInfo`` for ``Release`` builds. But Conan default settings (in the Conan *settings.yml* file), only have ``Debug`` and ``Release``
+defined. It is possible to modify the *settings.yml* file, and add those extra build types. Then you should create the ``hello`` package
+for those settings. And most existing packages, specially in central repositories, are built only for Debug and Release modes.
+
+An easier approach is to change the CMake configuration in Visual: go to the Menu -> CMake -> Change CMake Configuration. That should open
+the *CMakeSettings.json* file, and there you can change the ``configurationType`` to ``Release``:
+
+.. code-block:: text
+   :emphasize-lines: 4
+
+    {
+        "name": "x64-Release",
+        "generator": "Ninja",
+        "configurationType": "Release",
+        "inheritEnvironments": [ "msvc_x64_x64" ],
+        "buildRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\build\\${name}",
+        "installRoot": "${env.USERPROFILE}\\CMakeBuilds\\${workspaceHash}\\install\\${name}",
+        "cmakeCommandArgs": "",
+        "buildCommandArgs": "-v",
+        "ctestCommandArgs": ""
+    }
+
+Note that the above CMake code is only valid for consuming existing packages. If you are also creating a package, you
+would need to make sure the right CMake code is executed, please check https://github.com/conan-io/cmake-conan/blob/master/README.md
+
+Using tasks with tasks.vs.json
```

Comment:
I like this, but it shouldn't be in the "cmake.rst" file

Reply:
Moved to its own section